setup.rst

```diff
@@ -240,7 +240,7 @@ Windows
 For a quick guide to building you can read `this documentation`_ from Victor
 Stinner.
 
-**Python 3.6** and later can use Microsoft Visual Studio 2017.  You can download
+**Python 3.7** and later can use Microsoft Visual Studio 2017.  You can download
```

Comment:
I believe this could be updated to something like the following:

```suggestion
All current versions of Python can be built using Microsoft Visual Studio 2017
or later.  You can download
```

It might be good to have @zooba confirm that, though :)